src/web/templates/web/partials/modal/detail_event.htm

```diff
@@ -0,0 +1,32 @@
+<div class="container-fluid my-3">
+    {% if object.group %}
```

Comment:
I'm not sure why we made the `group` FK nullable, seems like it should always be required... but that's outside the scope of the PR.

Reply:
Yeah, I'd think that should be required. The template should reflect the model, so those so be resolved together.